src/ui/settings/ViewGiveFeedback.qml

```diff
@@ -334,6 +340,15 @@ Item {
                    Layout.fillWidth: true
                    Layout.preferredHeight: Theme.rowHeight
                    onClicked: stackview.pop(StackView.Immediate)
+                   anchors.horizontalCenter: parent.horizontalCenter
```

Comment:
We should remove this line or change it to use a Layout aligner - `Layout.alignment: Qt.AlignHCenter`